test/core/GnosisSafe.Storage.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from "chai";
+
+import hre from "hardhat";
+import "@nomiclabs/hardhat-ethers";
+import { getContractStorageLayout } from "../utils/storage";
+
+describe("GnosisSafe", async () => {
+  it("follows storage layout defined GnosisSafeStorage library", async () => {
+    const gnosisSafeStorageLayout = await getContractStorageLayout(
+      hre,
+      "GnosisSafeStorage"
+    );
+    const gnosisSafeMasterCopyStorageLayout = await getContractStorageLayout(
+      hre,
+      "GnosisSafe"
+    );
+
+    expect(JSON.stringify(gnosisSafeMasterCopyStorageLayout)).to.include(
+      JSON.stringify(gnosisSafeStorageLayout)
+    );
```

Comment:
Chai doesn't have a matcher to check if an array is part of another array, I decided to convert it to a string

Reply:
Can we add this as a comment?